### DISCUSSION TITLE
Number-range scoring

### DISCUSSION THREAD

**Discussion by mapeje** [mawoka-myblock/ClassQuiz#309] [Ideas]
Hi. Would it be possible to get points when you are closer to the correct answer? Like 0-100 when you more or less guess the number and the closer you are to the correct answer, the more points you get?

**Comment by mawoka-myblock** [comment]
That's a good idea!

**Reply by mapeje** [reply]
Unfortunately, I am no coder and cannot contribute to such feature, but it would be great if it could happen :) Thanks!
